### DISCUSSION TITLE
Any idea when you will release 1.5.8

### DISCUSSION THREAD

**Discussion by tnatschl** [bytedeco/javacpp#591] [General]
Hi!

I'm wondering when javacpp 1.5.8 will be released? I'm particularly interested in the new feature to be able to disable library caching (`"-Dorg.bytedeco.javacpp.cachelibraries=false"`).

Any information appreciated.

Thx and Kr

  -Thomas (Natschläger)

**Comment by saudet** [comment]
Probably sometime this fall. In the meantime, you can use the snapshots: http://bytedeco.org/builds/

**Comment by tnatschl** [comment]
Thank you for the timeline and the pointer to the snapshot.

**Reply by saudet** [reply]
JavaCPP 1.5.8 has been released! Enjoy
